scraper/twitter/utils.py

```diff
@@ -0,0 +1,45 @@
+
+
+def rate_tweet(tweet):
+    """
+    This method will return a coeficient for a tweet
+    following some rules
+
+    # rules
+    # - It should not be a reply
```

Comment:
I think if it's not a reply then, tweets like threads won't be take in case. For example if i do a thread with 5 tweets, only the first one would be use by this function, the 4 others won't be used

You can correct me if i'm wrong